README_zh.md

```diff
@@ -5,7 +5,7 @@
 
 中文版 ｜ [English](README.md)
 
-DaoCloud 是经认证的 K8s 服务提供商。目前 DCE 经 CNCF 认证可以完美支撑以下版本：
+DaoCloud 是经认证的 K8s 服务提供商。目前 DCE 经 CNCF 认证可以完美支持以下版本：
```

Comment:
All right, you can find some more entry point here:

https://github.com/DaoCloud/DaoCloud-docs/issues

https://github.com/DaoCloud/DaoCloud-docs/discussions

btw, thanks for your contribution.